src/js/jsx/PanelSet.jsx

```diff
@@ -269,11 +276,19 @@ define(function (require, exports, module) {
             this.getFlux().actions.preferences.setPreferences(nextState);
             headlights.logEvent("user-interface", "panel-toggle", panelName);
         },
+        
+        /**
+         * True if the current docuemnt is initialized and its panels should be displayed.
+         * 
+         * @return {Boolean}
+         */
+        _hasActiveDocument: function () {
+            return this.state.activeDocument && this.state.activeDocumentInitialized && this.state.documentIDs.size > 0;
+        },
 
         render: function () {
-            var documentIDs = this.state.documentIDs,
-                activeDocument = this.state.activeDocument,
-                hasActiveDocument = activeDocument && this.state.activeDocumentInitialized && documentIDs.size > 0;
+            var activeDocument = this.state.activeDocument,
```

Comment:
I find `_getActiveDocument` is confusing with `this.state.activeDocument`. The difference is that `_getActiveDocument` only returns the an initialized active document. Also, in the `render` function, un-initialized active document is accepted, but the panels will remain hidden. So, instead of changing the return value, I rename `_hasActiveDocument ` to `_isActiveDocumentInitialized`, which looks a little better. Please let me know if you still think we should use `_getActiveDocument` instead.